app/assets/javascripts/application.js

```diff
@@ -474,6 +474,12 @@ function set_fullscreen(element){
   });
 }
 
+function trunc(data, size){
```

Comment:
I'd add a tooltip so you can see the original data.
Maybe reuse trunc_with_tooltip and change it to get the '...' instead of changing this here?
also underscore,js that was recently added might have a functions for this.